hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleSchedule.java

```diff
@@ -47,48 +51,53 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 import static com.google.protobuf.ByteString.copyFrom;
-import static com.hedera.services.legacy.core.jproto.JKey.equalUpToDecodability;
+import static com.hedera.services.utils.MiscUtils.asOrdinary;
 import static com.hedera.services.utils.MiscUtils.asTimestamp;
 import static com.hedera.services.utils.MiscUtils.describe;
 import static com.swirlds.common.CommonUtils.hex;
 import static java.util.stream.Collectors.toList;
 
-
 public class MerkleSchedule extends AbstractMerkleLeaf implements FCMValue {
 	static final int MERKLE_VERSION = 1;
 
 	static final int NUM_ED25519_PUBKEY_BYTES = 32;
-	static final int UPPER_BOUND_MEMO_UTF8_BYTES = 1024;
 
 	static final long RUNTIME_CONSTRUCTABLE_ID = 0x8d2b7d9e673285fcL;
 	static DomainSerdes serdes = new DomainSerdes();
 
+	public static final Key UNUSED_GRPC_KEY = null;
 	public static final JKey UNUSED_KEY = null;
 	public static final EntityId UNUSED_PAYER = null;
+	public static final RichInstant UNRESOLVED_TIME = null;
 
-	private byte[] grpcTxn;
-	private String memo;
+	private Key grpcAdminKey = UNUSED_GRPC_KEY;
 	private JKey adminKey = UNUSED_KEY;
+	private String memo;
+	private boolean deleted = false;
+	private boolean executed = false;
 	private EntityId payer = UNUSED_PAYER;
 	private EntityId schedulingAccount;
 	private RichInstant schedulingTXValidStart;
 	private long expiry;
+	private RichInstant resolutionTime = UNRESOLVED_TIME;
+
+	private byte[] bodyBytes;
```

Comment:
I don't think that we need this property. These are the body bytes of the parent `scheduleCreate` TX